backend/module/eCampApi/test/Rpc/test.module.config.php

```diff
@@ -179,52 +189,50 @@
     ],
     'api-tools-rpc' => [
         'eCampApi\\V1\\Rpc\\Auth\\AuthController' => [
-            'service_name' => 'Auth',
             'http_methods' => [
                 0 => 'GET',
                 1 => 'POST',
+                2 => 'GET',
+                3 => 'POST',
+                4 => 'GET',
+                5 => 'POST',
```

Comment:
Laminas may be fine with this, but if you add a `merge` method to the Factory, this could maybe be avoided (but don't bother in case it's too much effort)

Reply:
I let laminas do the merging, i don't think it's necessary.